### ISSUE TITLE
Option to colorize Query Tool tab header based on server color (RM #4070)

### ISSUE THREAD

**Issue by dpage** [pgadmin-org/pgadmin4#2431]
***Issue migrated from Redmine: https://redmine.postgresql.org/issues/4070***
*Originally created by **Alex Decherd** at 2019-03-11 19:08:47 UTC.*

A check box option that, when true, would color the Query Tool tab headers with their configured server color.

Sometimes, when you have Query Tools opened for more than one server, it would be handy to be able to see which server a Query Tool tab is for at-a-glance.

**Comment by enriquein** [comment]
Thank you for this, it's extremely useful for me. I did notice that the colored circle only appears if the tab is not the active tab. If this was intentional, may I humbly ask if it's possible to make it optional to always show the colored circle?

**Comment by dev-hari-prasad** [comment]
> Thank you for this, it's extremely useful for me. I did notice that the colored circle only appears if the tab is not the active tab. If this was intentional, may I humbly ask if it's possible to make it optional to always show the colored circle?

Yes that would be super handy this would be a very good addition to the existing color dots. Happy to implement it if maintainers agree or I can make the addition to the existing PR which is already begin worked upon. @akshay-joshi @asheshv

**Comment by dpage** [comment]
Shipped in 9.16: "Added an option to colourize panel and tab headers based on the connected server's colour, making it easier to tell which server a tab is connected to at a glance." Closing as completed.
